src/app/domain/sharing/services/SharingService.ts

```diff
@@ -0,0 +1,75 @@
+import ReceiveSharingIntent from '@mythologi/react-native-receive-sharing-intent'
+
+import Minilog from 'cozy-minilog'
+
+import {
+  isAndroidFile,
+  isIOSFile,
+  ReceivedFile
+} from '/app/domain/sharing/models/ReceivedFile'
+
+export const sharingLogger = Minilog('🗃️ SharingService')
+const filesToUpload = new Map<string, ReceivedFile>()
```

Comment:
Do we really need this local state? Seems like `getFilesToUpload()` is never called outside of this file. And `SharingProvider` also stores this info in a reactive way.

The only methods that needs this is `hasFilesToUpload` but I'm wondering if we can move it in `SharingProvider`, can't we?